packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -702,11 +702,15 @@ test(`zcfSeat.getNotifier`, async t => {
   const notifier = zcfSeat.getNotifier();
   // These are different notifiers
   t.not(notifier, await E(userSeat).getNotifier());
+  t.deepEqual(await notifier.getUpdateSince(), {
+    updateCount: 2,
+    value: {},
+  });
```

Comment:
I don't think the notifier on the user's side should notify about allocations before the offer is made.